tests/headers/objc_generic.h

```diff
@@ -0,0 +1,12 @@
+// bindgen-flags: --objc-extern-crate -- -x objective-c
+// bindgen-osx-only
+@interface Base
+@end
```

Comment:
This test should probably just reuse the `objc_template.h` header rather than making a whole new one.